src/lib/components/Instances/InstanceTable.svelte

```diff
@@ -66,6 +67,29 @@
 		);
 		deleteInstance(id);
 	}
+
+	async function showLabels() {
+		// set project labels to true
+		project.show_labels = true;
+		const projectRes = await toggleProjectLabels(
+			project.id,
+			prompt,
+			project.show_labels,
+			project.metric_name
+		);
+		project.metric_name = projectRes.metric_name;
+		// update instances with labels = predictions
+		instances = await Promise.all(
```

Comment:
should we instead do this with a backend endpoint? This is a ton of fetch requests and a bit complicated. Instead, we can have one backend endpoint that does this in one operation maybe?